roles/edpm_network_config/tasks/nmstate_tool.yml

```diff
@@ -42,3 +61,17 @@
       ansible.builtin.include_role:
         name: "{{ lookup('ansible.builtin.env', 'EDPM_SYSTEMROLES', default='fedora.linux_system_roles') + '.network' }}"
       register: nmstate_result
```

Comment:
Have you tested this? I don't think nmstate_result would contain the failure details for errors configuring networking connection profiles.. In cases there are errors executing the role, ansible would stop at the task and it won't execute task from L71.

Reply:
If it reaches L71 then you've to assume it has configured the network. Though nmstate.returncode file would never have non zero value. But we can live with it I guess. There is probably no need to register and check the result.